### PR TITLE
Wrapped Filecoin documentation update

### PR DESCRIPTION
**Pull request by TkKonstantin**
Adding documentation about wrapped filecoin solutions

### CODE REVIEW

**Comment by filecorgi** [comment]
- I optimized some images for you! See the commit with the comment `Automatically optimized images [ciskip]` in this PR for details.
- Vuepress build was successful!
### `docs/about-filecoin/wrapped-filecoin.md`:
- This word is normally spelled as one.
  ```
  ... (coming soon wFIL) etc. # Custodial or non-custodial wrapping solution? Non...
                                             ^^^^^^^^^^^^^
  ```
- This word is normally spelled as one.
  ```
  ...ial or non-custodial wrapping solution? Non-custodial designs (renFIL, vFIL)...
                                             ^^^^^^^^^^^^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...on. Click to mint. You may need to wait several minutes for the target netwo...
                                             ^^^^^^^
  ```

File: docs/about-filecoin/wrapped-filecoin.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+- STACKS (coming soon wFIL)
+- KADENA (coming soon wFIL)
+- etc.
+
+## Custodial or non-custodial wrapping solution?
+
+- **Non-custodial designs** (renFIL, vFIL) are governed only by on-chain, immutable smart contracts. No single or collective entity has a master key that could force trades, transactions, or other operations to follow one direction or the other.
+- **Custodial designs** (wFIL, BFIL, eFIL, HFIL) are managed by a single third party. They may have services always have a counter-party risk from their operator but could have some other advantages like insurance, guaranteed backing, support, etc.
+
+## Wrapping your FIL process
```
Comment (johnnymatthews) [inline docs/about-filecoin/wrapped-filecoin.md:39]:
```suggestion
## How to wrap your FIL
```

File: docs/about-filecoin/wrapped-filecoin.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+There are several web-based applications for token wrapping. Here is an overview of the steps:
+
+1. **Choose your wrapped FIL type.** Depending on the chain where you want to use your wrapped FIL, different types are available.
+2. **Open the web app and connect your wallet.** Most apps require a web or mobile wallet.
+3. **Choose the amount.** Double-check the fee structure, which may vary by network and application.
+4. **Click to mint.** You may need to wait several minutes for the target network to confirm the transaction.
+
+For example, here’s how it works in RenBridge, a decentralized application (dApp) that allows users to mint real FIL (as well as BTC, ZEC, and more) on Ethereum as an ERC20 token (renFIL):
```
Comment (johnnymatthews) [inline docs/about-filecoin/wrapped-filecoin.md:48]:
```suggestion
Here’s how it works in RenBridge, a decentralized application (dApp) that allows users to mint real FIL on Ethereum as an ERC20 token:
```

File: docs/about-filecoin/wrapped-filecoin.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    </tr>
+    <tr>
+        <td>HFIL</td>
+        <td>Huobi</td>
+        <td>Decentralized</td>
+        <td>Huobi</td>
+        <td>Etherium</td>
+        <td>https://www.huobi.com</td>
+    </tr>
+</table>
```
Comment (johnnymatthews) [inline docs/about-filecoin/wrapped-filecoin.md:117]: Tables should follow the markdown syntax: https://www.markdownguide.org/extended-syntax/#tables

**Review by johnnymatthews — CHANGES_REQUESTED** [review]
This is a great PR! I've just added some questions and suggestions that need to be addressed before we can merge this.

File: docs/about-filecoin/wrapped-filecoin.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+Users can transact confidently because the wrapped token value is preserved by a smart contract that backs each token one-to-one with the underlying assets. This gives users greater flexibility — such as the ability to write smart contracts with assets across multiple networks without waiting for both chains to update, to create greater liquidity for decentralized finance, or to optimize block confirmation times or gas fees by choosing different networks.
+
+::: callout
+Although Filecoin tokens (FIL) cannot be directly used on Ethereum (ETH) or other networks due to different standards, **wrapped FIL** offer the ability to use FIL tokens 1:1 within other blockchain networks including Ethereum, Polygon, Solana, Celo, and more.
+:::
+
+## Where can I use wrapped FIL?
+
+Wrapped FIL is available in the following networks:
```
[the remark this answers is not in the document]
Comment (TkKonstantin) [reply docs/about-filecoin/wrapped-filecoin.md:20]:
```suggestion

Although Filecoin tokens (FIL) cannot be directly used on Ethereum (ETH) or other networks due to different standards, **wrapped FIL** offer the ability to use FIL tokens 1:1 within other blockchain networks including Ethereum, Polygon, Solana, Celo, and more.

## Where can I use wrapped FIL?

Wrapped FIL is available in the following networks:
```

Lets just remove the callout then to keep the original structure of the post

File: docs/about-filecoin/wrapped-filecoin.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+- Arbitrum (renFIL)
+- Celo (wFIL)
+- STACKS (coming soon wFIL)
+- KADENA (coming soon wFIL)
+- etc.
+
+## Custodial or non-custodial wrapping solution?
+
+- **Non-custodial designs** (renFIL, vFIL) are governed only by on-chain, immutable smart contracts. No single or collective entity has a master key that could force trades, transactions, or other operations to follow one direction or the other.
+- **Custodial designs** (wFIL, BFIL, eFIL, HFIL) are managed by a single third party. They may have services always have a counter-party risk from their operator but could have some other advantages like insurance, guaranteed backing, support, etc.
```
[the remark this answers is not in the document]
Comment (TkKonstantin) [reply docs/about-filecoin/wrapped-filecoin.md:37]:
```suggestion
- **Custodial designs** (wFIL, BFIL, eFIL, HFIL) are managed by a single third party. Centralized services always have a counter-party risk from their operator but could provide other advantages like insurance, guaranteed backing, support, etc.
```

I thought, I edited this part in the submitted version.

[the remark this answers is not in the document]
Comment (TkKonstantin) [reply docs/about-filecoin/wrapped-filecoin.md:37]:
```suggestion
- **Custodial designs** (wFIL, BFIL, eFIL, HFIL) are managed by a single third party. They may have services always have a counter-party risk from their operator but could have some other advantages like insurance, guaranteed backing and support.
```

File: docs/about-filecoin/wrapped-filecoin.md (hunk 1 above)
Comment (TkKonstantin) [reply docs/about-filecoin/wrapped-filecoin.md:39]:
```suggestion
## How to wrap your FIL?
```

Sounds good to me. 
@mishmosh what do you think?

File: docs/about-filecoin/wrapped-filecoin.md (hunk 2 above)
Comment (TkKonstantin) [reply docs/about-filecoin/wrapped-filecoin.md:48]:
```suggestion
Here’s how it works in RenBridge, a decentralized application (dApp) that allows users to mint real FIL into renFIL on Ethereum and other networks:
```

That looks better to me.

[the remark this answers is not in the document]
Comment (TkKonstantin) [reply docs/about-filecoin/wrapped-filecoin.md:48]: RenBridge was one of the most straightforward tools to use with the most target networks available. We thought it was important to showcase how easy wrapping could be on one of the examples with the visual explanations.

File: docs/about-filecoin/wrapped-filecoin.md (hunk 3 above)
Comment (TkKonstantin) [reply docs/about-filecoin/wrapped-filecoin.md:117]: I actually used the same table format as in https://github.com/filecoin-project/filecoin-docs/blob/master/docs/about-filecoin/filecoin-compared-to.md, just updated it to my needs. Will shortly update it to the markdown syntax

Comment (TkKonstantin) [reply docs/about-filecoin/wrapped-filecoin.md:117]:
```suggestion
| **Name** | **Project** | **Infrastructure** | **Custodian** | **Main Blockchain**              | **Link**                            |
|----------|-------------|--------------------|---------------|----------------------------------|-------------------------------------|
| renFIL   | renBridge   | Decentralized      | Decentralized | Etherium, Binance, Polygon       | https://bridge\.renproject\.io/mint |
| wFIL     | WRAPPED     | Centralized        | Anchorage     | Etherium                         | https://www\.wrapped\.com/          |
| BFIL     | Binance     | Centralized        | Binance       | Etherium via Binance Token Canal | https://www\.binance\.com/          |
| vFIL     | Venus       | Decentralized      | Decentralized | Binance Smart Chain              | https://www\.venus\.io/             |
| eFIL     | Gemini      | Decentralized      | Gemini        | Etherium                         | https://www\.gemini\.com/           |
| HFIL     | Huobi       | Decentralized      | Huobi         | Etherium                         | https://www\.huobi\.com             |
```

This works better?

File: docs/about-filecoin/wrapped-filecoin.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (TkKonstantin) [reply docs/about-filecoin/wrapped-filecoin.md:37]:
```suggestion
- **Custodial designs** (wFIL, BFIL, eFIL, HFIL) are managed by a single third party. Centralized services always have a counter-party risk from their operator but could provide other advantages like insurance, guaranteed backing and support.
```
Incorporating next comment

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

### `docs/about-filecoin/wrapped-filecoin.md`:
- This word is normally spelled as one.
  ```
  ... (coming soon wFIL) etc. # Custodial or non-custodial wrapping solution? Non...
                                             ^^^^^^^^^^^^^
  ```
- This word is normally spelled as one.
  ```
  ...ial or non-custodial wrapping solution? Non-custodial designs (renFIL, vFIL)...
                                             ^^^^^^^^^^^^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...on. Click to mint. You may need to wait several minutes for the target netwo...
                                             ^^^^^^^
  ```

File: docs/about-filecoin/wrapped-filecoin.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (johnnymatthews) [reply docs/about-filecoin/wrapped-filecoin.md]: That's fair, thanks for clarifying!

**Review by johnnymatthews — APPROVED** [review]
Looks good! Thanks for making those changes.

File: docs/about-filecoin/wrapped-filecoin.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
+---
+title: Wrapped Filecoin
+description: Wrapped tokens allow tokens from one blockchain to be used natively on another chain. 
+---
+
+# Wrapped Filecoin
+
+## What is a wrapped token?
+
```
Comment (johnnymatthews) [inline docs/about-filecoin/wrapped-filecoin.md:9]:
```suggestion
# Wrapped Filecoin

```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

### `docs/about-filecoin/wrapped-filecoin.md`:
- This word is normally spelled as one.
  ```
  ... (coming soon wFIL) etc. # Custodial or non-custodial wrapping solution? Non...
                                             ^^^^^^^^^^^^^
  ```
- This word is normally spelled as one.
  ```
  ...ial or non-custodial wrapping solution? Non-custodial designs (renFIL, vFIL)...
                                             ^^^^^^^^^^^^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...on. Click to mint. You may need to wait several minutes for the target netwo...
                                             ^^^^^^^
  ```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

### `docs/about-filecoin/wrapped-filecoin.md`:
- This phrase is duplicated. You should probably use “wrapped FIL” only once.
  ```
  ... Solana, Celo, and more. # Where to use wrapped FIL Wrapped FIL is available...
                                             ^^^^^^^^^^^^^^^^^^^^^^^
  ```
- This word is normally spelled as one.
  ```
  ...ADENA (coming soon wFIL) # Custodial or non-custodial wrapping solution Non-...
                                             ^^^^^^^^^^^^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...on. Click to mint. You may need to wait several minutes for the target netwo...
                                             ^^^^^^^
  ```

**Pull request merged**